tests/test_checks.py

```diff
@@ -0,0 +1,34 @@
+# import arcgis
+
+from auditor import credentials
+from auditor import checks
+from auditor.auditor import Auditor
+
+# @pytest.fixture
+# def agol_item():
+#     import arcgis
+#     org = arcgis.gis.GIS(credentials.ORG, credentials.USERNAME, credentials.PASSWORD)
+#     item = org.content.get('d2ea8eb22aa64d22800bc370bb1b128b')
+#     return item
+
+# @pytest.fixture
+# def metatable_dict():
+#     fake_metatable = {
+#         'd2ea8eb22aa64d22800bc370bb1b128b':
+#             ['SGID.WATER.Stations',
+#             'Utah AGOL Upload Test',
+#             'SGID',
+#             'y']
+#         }
+
+#     return fake_metatable
```

Comment:
**suggestion**: I'd just remove this code if it's not being used.

Reply:
This is all part of my adventures with pytest. I'll get it sorted out as I get more accomplished there.